coins/monero/src/tests/seed.rs

```diff
@@ -0,0 +1,22 @@
+use crate::wallet::seed::{Seed, LanguageName};
+
+#[test]
+fn test_classic() {
+  // keys
+  let spend = "c0af65c0dd837e666b9d0dfed62745f4df35aed7ea619b2798a709f0fe545403";
+  let _view = "513ba91c538a5a9069e0094de90e927c0cd147fa10428ce3ac1afd49f63e3b01";
+
+  // seed string
+  let mut seed = String::from("washing thirsty occur lectures tuesday fainted toxic adapt ");
```

Comment:
We need to test all languages, and their trimmed variants (for languages supporting trimming).

Reply:
Thought we would add all the tests when I switch back to tests.

Reply:
If we're adding new features, they should be tested.